spec/models/work_view_stat_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'spec_helper'
+
+RSpec.describe WorkViewStat, type: :model do
+  let(:work_id) { "199" }
+  let(:user_id) { 123 }
+  let(:date) { DateTime.new }
+  let(:work_stat) { described_class.create(views: "25", date: date, work_id: work_id_id, user_id: user_id) }
+
+  it "has attributes" do
+    expect(work_stat).to respond_to(:views)
+    expect(work_stat).to respond_to(:date)
+    expect(work_stat).to respond_to(:work_id)
+    expect(work_stat.work_id).to eq("199")
+    expect(work_stat.date).to eq(date)
+    expect(work_stat.views).to eq(25)
+    expect(work_stat.user_id).to eq(user_id)
+  end
+
+  describe "#get_float_statistics" do
+    let(:dates) {
+      ldates = []
+      4.downto(0) { |idx| ldates << (Date.today - idx.day) }
+      ldates
+    }
+    let(:date_strs) {
+      dates.map { |date| date.strftime("%Y%m%d") }
+    }
+
+    let(:view_output) {
+      [[statistic_date(dates[0]), 4], [statistic_date(dates[1]), 8], [statistic_date(dates[2]), 6], [statistic_date(dates[3]), 10]]
+    }
+
+    # This is what the data looks like that's returned from Google Analytics (GA) via the Legato gem
+    # Due to the nature of querying GA, testing this data in an automated fashion is problematc.
+    # Sample data structures were created by sending real events to GA from a test instance of
+    # Scholarsphere.  The data below are essentially a "cut and paste" from the output of query
+    # results from the Legato gem.
+    let(:sample_pageview_statistics) {
+      [
+        OpenStruct.new(date: date_strs[0], pageviews: 4),
+        OpenStruct.new(date: date_strs[1], pageviews: 8),
+        OpenStruct.new(date: date_strs[2], pageviews: 6),
+        OpenStruct.new(date: date_strs[3], pageviews: 10),
+        # OpenStruct.new(date: date_strs[4], pageviews: 2)
```

Comment:
Why's this one commented out? Was it also commented in the file you used as a pattern for this?